addons/mrp_account/models/product.py

```diff
@@ -79,7 +79,7 @@ def _compute_bom_price(self, bom, boms_to_recompute=False, byproduct_bom=False):
             duration_expected = (
                 opt.workcenter_id._get_expected_duration(self) +
                 opt.time_cycle * 100 / opt.workcenter_id.time_efficiency)
-            total += (duration_expected / 60) * opt._total_cost_per_hour()
+            total += (duration_expected / 60) * opt._total_cost_per_hour() * bom.product_qty
```

Comment:
This is only correct if the workcenter `capacity` is 1.

Reply:
Done! Thanks for the review ;)